forge-ai/src/main/java/forge/ai/CreatureEvaluator.java

```diff
@@ -247,6 +247,14 @@ else if (c.hasKeyword(Keyword.WITHER)) {
             value -= subValue(20, "upkeep-dmg");
         } 
 
+        // evaluate abilities
+        // TODO: expand this for other scenarios where an ability may influence the priority
+        for (SpellAbility sa : c.getSpellAbilities()) {
```

Comment:
I don't think we can/should tweak the Evaluator like that...
You might sometimes get better results on removal decisions, but at the same time you've now taught the AI it should rather play a vanilla 1/1 than _Sakura-Tribe Elder_ when given the choice

Reply:
Ah, indeed! Hmm, maybe this should then be limited to cards that are already on the battlefield and are controlled by the opponent? Then again, there will indeed be cases when there will be a need to prioritize it higher with something... Maybe dropping the idea is better then? 🤔 Do you have any ideas?

Reply:
you could override `evaluateCreature` with a boolean or API param so you can check the context
or just another helper function that can filter, though finding all the right places to apply it might be a bit unpleasant

Reply:
Gotcha! Yeah, I'll revert the current approach for now and I'll think a bit on how to implement it better, possibly as a separate PR since this one is getting pretty fat by now XD